snd-server/models/request_forms.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+from fastapi.security import OAuth2PasswordRequestForm
+from fastapi.param_functions import Form
+
+
+class SignUpRequestForm(OAuth2PasswordRequestForm):
+    def __init__(
+        self,
+        grant_type: str = Form(default=None, regex="password"),
+        username: str = Form(),
+        password: str = Form(),
+        email: str = Form(),
+        scope: str = Form(default=""),
+        client_id: Optional[str] = Form(default=None),
+        client_secret: Optional[str] = Form(default=None),
```

Comment:
Do we really need form encoding format for this? Would probably be easier to just keep using json for everything, I'm not planning on submitting requests from client via html form actions :)